core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -79,8 +79,8 @@ oppia.factory('UrlInterpolationService', [
         var INVALID_VARIABLE_REGEX = /(<{2,})(\w*)(>{2,})/;
 
         // Parameter values can only contain alphanumerical characters, spaces,
-        // hyphens, underscores, or periods.
-        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.])+$/;
+        // hyphens, underscores, periods or the equal to symbol.
+        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.]|=)+$/;
```

Comment:
The thread id does contain two == sign in the middle which is why it is failing. I logged this value and checked.

Should we revert back the earlier method of not using URL interpolation and using concatenation?